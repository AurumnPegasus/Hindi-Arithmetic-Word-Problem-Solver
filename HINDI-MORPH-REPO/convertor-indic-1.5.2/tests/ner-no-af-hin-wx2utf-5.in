<Sentence id="5"> 
1	((	NP	<fs etype='timex' esubtype='timeOfDay'>
1.1	SAma	unk	<fs etype='timex' esubtype='timeOfDay'>
	))
2	meM	unk
3	((	NP	<fs etype='timex' esubtype='time' value='7:20'>
3.1	((	NP	<fs etype='numx' value='7.0'>
3.1.1	7	unk	<fs etype='numx' value='7.0'>
	))
3.2	bajakara	unk
3.3	((	NP	<fs etype='numx' value='20.0'>
3.3.1	20	unk	<fs etype='numx' value='20.0'>
	))
3.4	minata	unk	<fs etype='timex' esubtype='timeMeasure' Unit='Minute' 'value='1.0'>
	))
4	para	unk
5	sUrya	unk
6	((	NP	<fs etype='timex' esubtype='durationOfEra'>
6.1	uwwarAyaNa	unk	<fs etype='timex' esubtype='durationOfEra'>
	))
7	meM	unk
8	praveSa	unk
9	karegA	unk
10	.	unk
</Sentence>
