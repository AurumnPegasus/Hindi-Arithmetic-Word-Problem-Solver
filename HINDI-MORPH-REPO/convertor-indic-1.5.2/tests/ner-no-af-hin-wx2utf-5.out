<Sentence id="5"> 			
1	((	NP	<fs  esubtype='timeOfDay'  etype='timex'>
1.1	शाम	unk	<fs  esubtype='timeOfDay'  etype='timex'>
	))
2	में	unk	
3	((	NP	<fs  esubtype='time'  etype='timex'  value='7:20'>
3.1	((	NP	<fs  etype='numx'  value='7.0'>
3.1.1	7	unk	<fs  etype='numx'  value='7.0'>
	))
3.2	बजकर	unk	
3.3	((	NP	<fs  etype='numx'  value='20.0'>
3.3.1	20	unk	<fs  etype='numx'  value='20.0'>
	))
3.4	मिनट	unk	<fs  esubtype='timeMeasure'  etype='timex'  value='1.0'  Unit='Minute'>
	))
4	पर	unk	
5	सूर्य	unk	
6	((	NP	<fs  esubtype='durationOfEra'  etype='timex'>
6.1	उत्तरायण	unk	<fs  esubtype='durationOfEra'  etype='timex'>
	))
7	में	unk	
8	प्रवेश	unk	
9	करेगा	unk	
10	.	unk	
</Sentence>			
